src/app/chat/view.nim

```diff
@@ -75,7 +85,9 @@ QtObject:
       result = self.chats.chats.findById(channel)
     else:
       self.model.join(channel)
-      result = self.chats.addChatItemToList(ChatItem(id: channel, name: channel))
+      randomize()
+      let randomColorIndex = rand(channelColors.len - 1)
```

Comment:
I think the chat colors are deterministic. I saw the color being part of the message signal

Reply:
They're not
I found the code in status-react. I was actually surprised. 
https://github.com/status-im/status-react/blob/develop/src/status_im/chat/models.cljs#L90

I tried opening #test in mobile and my simulator and had different colors.